src/jit/compiler.h

```diff
@@ -8338,7 +8339,12 @@ XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
         // PInvoke transitions inline (e.g. when targeting CoreRT).
         bool ShouldUsePInvokeHelpers()
         {
+#ifndef PLATFORM_UNIX
```

Comment:
Instead of hardcoding policy like this in the JIT, it would be better to set the `JIT_FLAG_USE_PINVOKE_HELPERS` flag on the EE side.

Reply:
Moved to EE side. Thanks!